app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -349,7 +349,7 @@ public void onPageStop(GeckoSession session, boolean success) {
                                 isSecure = true;
                             }
 
-                            callback.onProgress(100);
+                            callback.onProgress(99);
```

Comment:
Is this left over from a previous patch or is this needed?

Reply:
We set WebView progress to 99 before calling pageFinished, so I think we should do the same here.

Reply:
you're right we don't need this since the max we can set it to in onProgress is 99 anyway.